.travis.yml

```diff
@@ -28,7 +27,7 @@ before_script:
   - ./travisci/bin/ci/setup.sh
   - composer install
 
-script: vendor/bin/phpunit --debug
+script: vendor/bin/phpunit --enforce-time-limit --debug
```

Comment:
This depends on the test size annotation which will default to `@small`  see https://phpunit.de/manual/current/en/risky-tests.html#risky-tests.test-execution-timeout